src/core/components/InstallButton/index.js

```diff
@@ -72,6 +84,32 @@ export class InstallButtonBase extends React.Component {
           {i18n.gettext('Install Theme')}
         </Button>
       );
+    } else if (addon.type === ADDON_TYPE_OPENSEARCH) {
```

Comment:
Shoot, I thought I disabled it but maybe that got lost when I moved more of the logic into the button? Yes, it should be disabled, I'll do that.